src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  justify-items: center;
+}
+
+.stars_star {
+  background-position-x: center;
+  background-position-y: center;
+  margin-right: 4px;
+
+  @include star-size;
+
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 > :nth-child(n),
+.stars--4 > :nth-child(-n + 4),
+.stars--3 > :nth-child(-n + 3),
+.stars--2 > :nth-child(-n + 2),
+.stars--1 > :nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
```

Comment:
delete background-repeat property, you've already set it to the stars__star class